README.md

```diff
@@ -60,9 +60,11 @@ You can also query the API from the command line, using `curl`:
 &emsp;List the contents of the `intake` table: \
 &emsp;&emsp;`curl -k https://localhost:443/list?table=intake` \
 &emsp;Post the `sample.xlsx` file to the `/load` endpoint: \
-&emsp;&emsp;`curl -k -X POST "file=@resources/sample.xlsx" https://localhost:443/load` \
+&emsp;&emsp;`curl -k -X --form POST "file=@resources/sample.xlsx" https://localhost:443/load` \
```

Comment:
I think the --form has to go after the POST.  in the current format I get errors taht it can't resolve host POST and can't resolve host resources.

Reply:
Looks good, should probably have some error catching for the failed JSONs.  It was just outputting the traceback

Reply:
Good catch, I'll add some error catching